collectors/python.d.plugin/anomalies/README.md

```diff
@@ -35,18 +35,26 @@ Then, as the issue passes, the anomaly probabilities should settle back down int
 ## Requirements
 
 - This collector will only work with Python 3 and requires the packages below be installed.
+- Typically you will not need to do this, but, if needed, to ensure Python 3 is used you can add the below line to the `[plugin:python.d]` section of `netdata.conf`
+
+```yaml
+[plugin:python.d]
+	# update every = 1
+	command options = -ppython3
+```
```

Comment:
GitHub is showing some error here, I think because you labeled this codeblock as `yaml` and/or the whitespace? Maybe this suggestion will fix it.

````suggestion
```conf
[plugin:python.d]
    # update every = 1
    command options = -ppython3
```
````